src/features/events_list/atoms/eventSortingConfig.ts

```diff
@@ -0,0 +1,43 @@
+import { action, atom } from '@reatom/framework';
+import { AppFeature } from '~core/app/types';
+import { configRepo } from '~core/config';
+import type { EventsListFeatureConfig } from '~core/config/types';
+import type { SortByMCDAScoreConfig } from '~utils/mcda_sort/sortByMCDAScore';
+
+const DEFAULT_SORT_CONFIG: EventSortConfig = {};
+
+export type EventSortConfig = {
+  order?: 'asc' | 'desc' | undefined;
```

Comment:
Do we need `undefined` if we already have optional prop `?:` ?

Reply:
Fixed. Removed a bunch of other conditional fields in types, added config validation instead